kbase-extension/static/narrative_paths.js

```diff
@@ -394,7 +391,9 @@ require.config({
     map: {
         '*': {
             css: 'ext_components/require-css/css',
+            'jquery-dataTables': 'datatables.net-bs'
```

Comment:
in a near-future PR, perhaps we can do a search-n-replace so we don't have several different ways of referring to datatables.

Reply:
Sure. I'd like to get this merged -> truss first, and move forward with the DU deploy. Then time for cleanup! Nearly all of these dependencies are VERY out of date, some by multiple major versions. I think dependabot is going to have an aneurysm.